src/components/school/index/data/author.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Why not use a JSON structure like : 
```
{
  "author": {
    // ...
    "articles": [
      "get-started", "your-first-service", ...
    ]
  }
}
```
To avoid value duplication ?

Reply:
Looks like Thomas is missing here too